src/store/reducers/describe.ts

```diff
@@ -42,6 +45,18 @@ const describe = function z(
     }
 };
 
+// Consumers selectors
+const selectConsumersNames = (state: any, path: string): string[] =>
+    state.describe.data[path]?.PathDescription?.PersQueueGroup?.PQTabletConfig?.ReadRules || [];
+
+export const selectConsumers = createSelector(selectConsumersNames, (names) => {
+    const consumers = names.map((name) => {
+        return {name};
+    });
+
+    return consumers as IConsumer[];
```

Comment:
why cast?

Reply:
It was my way to determine function return type)

As I now understand, it’s could be done differently